TOOLS/osxbundle.py

```diff
@@ -80,7 +80,7 @@ def main():
 
     if options.deps:
         print("> bundling dependencies")
-        print(sh(" ".join(["TOOLS/dylib-unhell.py", target_binary(binary_name)])))
+        print(sh(" ".join(["python3", "TOOLS/dylib-unhell.py", target_binary(binary_name)])))
```

Comment:
If you changed the interpreter in dylib-unhell.py to python3 this won't be necessary.

Reply:
Thanks. correct, just want to make sure the reviewer could find this line easily. Let me revert this

Reply:
Fixed. Please double check. Thanks.